docs/faq.rst

```diff
@@ -130,3 +130,28 @@ How do I shutdown a KSQL environment?
    .. code:: bash
 
        $ confluent destroy
+
+========================================================
+Can KQL connect to a Apache Kafka cluster over SASL_SSL?
+========================================================
+
+Yes! Internally, KSQL uses standard Kafka Consumers and Producers, which you
+can configure to connect to a secure Kafka cluster as you would for any app.
+
+For example, adding the following entries, to the property file you use to start
```

Comment:
suggest rewrite as: `For example, you can add the following entries to the KSQL properties file (ksqlserver.properties). This configuration enables KSQL to connect to a secure Kafka cluster using _PLAIN_ SASL (other options include GSSAPI / Kerberos), where the SSL certificates have been signed by a CA trusted by the default JVM trust store:`

Reply:
much better!